datafusion/common/src/scalar/mod.rs

```diff
@@ -4563,6 +4563,7 @@ impl_try_from!(UInt8, u8);
 impl_try_from!(UInt16, u16);
 impl_try_from!(UInt32, u32);
 impl_try_from!(UInt64, u64);
+impl_try_from!(Float16, f16);
```

Comment:
Since you add this here - should there be also `impl_scalar!(f16, Float16);` at https://github.com/Jefffrey/datafusion/blob/bfb9e92ec0235e80c29e1005a379db6d5f3475d3/datafusion/common/src/scalar/mod.rs#L4408 ?

Reply:
Good point, added